.gitlab-ci.yml

```diff
@@ -4,15 +4,12 @@ include:
 # SETUP
 
 variables:
-  CURRENT_CI_IMAGE: "5"
+  CURRENT_CI_IMAGE: "6"
   CI_IMAGE_DOCKER: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/dd-sdk-android:$CURRENT_CI_IMAGE
   GIT_DEPTH: 5
 
   DD_SERVICE: "dd-sdk-android"
-  DD_ENV_TESTS: "ci"
-  DD_INTEGRATION_JUNIT_5_ENABLED: "true"
   DD_CIVISIBILITY_ENABLED: "true"
-  DD_INSIDE_CI: "true"
```

Comment:
I would propose to remove only `DD_INTEGRATION_JUNIT_5_ENABLED` variable, other two seems not to be related to the change made.

Reply:
Sure, reverted the extra changes